### ISSUE TITLE
[Callout Popover] Information accessibility

### ISSUE THREAD

**Issue by jeremie-lucca** [LuccaSA/lucca-front#2465]
![image](https://github.com/LuccaSA/lucca-front/assets/25581936/de2ab978-b241-474f-a3b2-f88d46413252)

The numeric value should have an hidden text associated for screen readers like: 
- 2 errors
- 2 warnings
- 2 successes
- 2 informations

`{{buttonLabel}}<span class="u-mask"> errors</span>`
